src/python/pants/base/deprecated.py

```diff
@@ -9,6 +9,7 @@
 
 from packaging.version import InvalidVersion, Version
 
+from pants.option.scope import GLOBAL_SCOPE
```

Comment:
I gave you a bad suggestion. This now introduces a dependency cycle between `option` and `base`. You should stick with the magic string in this file, but it's good you changed the other.